src/openrct2-ui/Ui.cpp

```diff
@@ -55,7 +56,15 @@ int main(int argc, const char** argv)
         {
             // Run OpenRCT2 with a UI context
             auto env = to_shared(CreatePlatformEnvironment());
-            auto audioContext = to_shared(CreateAudioContext());
+            std::shared_ptr<IAudioContext> audioContext;
+            try
+            {
+                audioContext = to_shared(CreateAudioContext());
+            }
+            catch (SDLException&)
+            {
+                audioContext = to_shared(CreateDummyAudioContext());
```

Comment:
Failure to create an audio context should be reported to the user as a warning

Reply:
So it should be `catch (const SDLException&)` ?

And would the warning use the log_warning macro? Or is this a pop up type warning?

Reply:
I meant https://en.cppreference.com/w/cpp/error/exception/what

Reply:
It's always nicer to the user to have the UI error in the UI, but `log_warning` will suffice if it's problematic to have it UI.